Src/PRuntimes/PSymbolicRuntime/pom.xml

```diff
@@ -34,7 +120,7 @@
               <artifactId>pjbdd</artifactId>
               <version>1.0.10-9</version>
               <packaging>jar</packaging>
-              <file>lib/pjbdd-v1.0.10-9.jar</file>
+              <file>${pjbdd.jarpath}</file>
```

Comment:
Can I request you to upgrade to the latest version of PJBDD as well. So we all the changes in one PR. thanks!

Reply:
Updated pjbdd to the latest release from https://www.sosy-lab.org/ivy/org.sosy_lab/pjbdd/